examples/appropriate-images/src/components/appropriate-image.js

```diff
@@ -0,0 +1,11 @@
+import { scopeAppropriateImage } from '@mapbox/appropriate-images-react';
+import imageConfig from '../img/dist/image.config.json'; // eslint-disable-line
+
+// See https://github.com/mapbox/appropriate-images-react#appropriateimage
+// The required prop is `imageId`, which must correspond to a key in the
+// imageConfig.
+const AppropriateImage = scopeAppropriateImage(imageConfig, {
+  transformUrl: (url) => require(`../img/dist/${url}`)
```

Comment:
You might need to add a `.default` to the end of this, similar to https://github.com/mapbox/batfish/pull/335/files#diff-4bc8782d65ced0f9881ace77188d0f7bL23